### PR TITLE
Refractor bottom nav

### PR DESCRIPTION
**Pull request by boldCM**
Refractored BottomNav,
Changed Favicon,
And changed href's to "Link to=" for react-router. Now Storybook is throwing this error:
```Invariant failed: You should not use <Link> outside a <Router>```
I'm using Router in my App.js and in previous projects we used the Link in other files without the router. Localhost is deploying as it should- only Storybook is throwing this error.
Any ideas on why?
Should I implement the router now on every file I use it?

### CODE REVIEW

**Comment by ncbecker** [comment]
To avoid the storybook error try to use the MemoryRouter in Storybooks preview.js like so:

```javascript
import GlobalStyle from "../src/GlobalStyle";
import { MemoryRouter } from "react-router-dom";

export const decorators = [
  (Story) => (
    <>
      <MemoryRouter>
        <GlobalStyle />
        <Story />
      </MemoryRouter>
    </>
  ),
];

export const parameters = {
  actions: { argTypesRegex: "^on[A-Z].*" },
  layout: "fullscreen",
};
```

File: client/src/components/BottomNav/BottomNav.js [hunk 1]
```diff
@@ -24,74 +23,22 @@ const BottomNavBar = styled.div`
...
-  bottom: 0;
-  right: 0;
-  margin-right: 11px;
-  margin-bottom: 9px;
-  position: fixed;
-  z-index: 1;
-`;
-
-const HomeButton = styled.a`
+const HomeButton = styled(Link)`
```
Comment (ncbecker) [inline client/src/components/BottomNav/BottomNav.js:26]: Nice to extend styling of the Link component 👍

File: client/src/components/BottomNav/MenueButton.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { useState } from "react";
+import DiamondButton from "../../assets/mainButton.svg";
+import Overlay from "../../pages/OverlayMenue/Overlay";
+
+const MainButton = styled.button`
+  background: none;
+  border: none;
```
Comment (ncbecker) [inline client/src/components/BottomNav/MenueButton.js:9]: I also like to set `outline: none` to get rid of the ugly blue outline when the button is focused/clicked

**Review by ncbecker — APPROVED** [review]
Nice work!

File: client/src/components/BottomNav/MenueButton.js (hunk 2 above)
Comment (boldCM) [reply client/src/components/BottomNav/MenueButton.js:9]: thanks :)
Me too- I did that in my Globalstyle.js :)

**Pull request merged**